build.gradle

```diff
@@ -54,8 +54,9 @@ dependencies {
     compile 'com.google.guava:guava:18.0'
     compile 'com.squareup.okhttp:okhttp:2.1.0-RC1'
     compile 'com.squareup.okhttp:okhttp-urlconnection:2.1.0-RC1'
-    compile 'com.github.kevinsawicki:http-request:5.6'
+    compile 'com.github.kevinsawicki:http-request:6.0'
```

Comment:
When will the release branch be merged into the develop branch? http-request is not used anymore in the release branch afaik.